modules/virt-creating-vm-snapshot-cli.adoc

```diff
@@ -64,11 +64,13 @@ $ kubectl wait my-vm my-vmsnapshot --for condition=Ready
 +
 [NOTE]
 ====
-Online snapshots have a default time deadline of five (`5`) minutes. If the snapshot does not complete successfully in five minutes, the status is set to `failed`. Afterwards, the file system will be thawed and the VM unfrozen but the status remains `failed` until you delete the failed snapshot image.
+Online snapshots have a default time deadline of five minutes (`5m`). If the snapshot does not complete successfully in five minutes, the status is set to `failed`. Afterwards, the file system will be thawed and the VM unfrozen but the status remains `failed` until you delete the failed snapshot image.
 
-To change the default time deadline, add the `FailureDeadline` attribute to the VM snapshot spec with the time (in minutes) that you want to specify before the snapshot operation times out.
+To change the default time deadline, add the `FailureDeadline` attribute to the VM snapshot spec with the time designated in minutes (`m`) that you want to specify before the snapshot operation times out.
```

Comment:
same comment in the other PR: I think you can add here after `in minutes ('m') ` add `or in seconds ('s')`

Reply:
Done